neo/rawio/blackrockrawio.py

```diff
@@ -311,70 +325,89 @@ def _parse_header(self):
 
         # This requires nsX to be parsed already
         # Needs to be called when no nsX are available as well in order to warn the user
+        # TODO QUESTION michael and julia
+        # do we need to do it for all nsx ?
```

Comment:
In principle yes. However in our use cases it is max. 2 nsx signals and 1 nev, but I think we should implement the general case.